src/resdk/resolwe.py

```diff
@@ -200,6 +200,18 @@ def version_check(self):
                 "Warning: unable to read the minimal supported version from the server."
             )
 
+    def version_output(self):
+        """Output the version of the server modules."""
+        url = urljoin(self.url, "/about/versions")
```

Comment:
Please add `/about/versions` as constant. See `MIMIMAL_SUPPORTED_VERSION_POSTFIX` as example (although its naming  is terrible).